docs/developing-flows/documenting-flows.md

```diff
@@ -0,0 +1,115 @@
+- Groups can be used to identify discrete sections of the flows.
+- Moving commonly used parts into subflows can help reduce the visual complexity of the flow.
+- More complete documentation can be added at the node, group or tab level
+
+
+### Laying out flows
+
+The [flow structure](flow-structure) section of this guide looked at how to arrange the logical components of your flows. This section considers the visual appearance of the flow layout.
+
+The goal is to make it easy to follow the flow without having to jump around the workspace or have to follow multiple wires that cross each other and appear tangled.
+
+The approach that gives the greatest legibility is to keep each unit of processing on a single horizontal line wherever possible. The editor's default behaviour of snapping nodes to a grid on the tab helps keep them aligned.
+
+<div class="figure">
+    <img src="./images/node-arrangement-sample.png" alt="Placeholder caption"/>
+    <p class="caption">Placeholder caption</p>
+</div>
+
+If there is a node that has more than one output port, aligning the branched flow vertically makes it easy to compare and contrast the flows.
+
+<div style="width: 600px" class="figure">
+  <img src="images/placeholder.png" alt="Link nodes">
+  <p class="caption">Placeholder for image: Cut above image in half - move Power Usage Calculation Service half here</p>
+</div>
+
+When a flow gets too long, arranging some nodes vertically can be used to good effect. In the following figure, some of the nodes are arranged vertically to imply a relationship between them. It is easier to understand the nature of the overall flow if it is visually obvious what smaller sections it is comprised of and how they relate to each other.
+
+<div class="figure">
+    <img src="./images/node-vertical-arrangement.png" alt="Placeholder caption"/>
+    <p class="caption">Placeholder caption</p>
+</div>
+
+In some cases, these smaller sections may be candidates for moving to subflows that will reduce the visual complexity of the flow. That is particular true if that smaller section could be reused elsewhere in the flows.
+
+### Naming nodes
+
+Most nodes have a `name` property that can be used to customise the label they display in the workspace. This should be used to properly label the key points of a flow.
+
+For example, if a Change node has a single rule that sets `msg.payload` to the current time, its default label will be `set msg.payload`. That helps somewhat, but it doesn't reveal the full purpose of the node. A name of `Get current time` would be much clearer.
+
+There is a balance to be considered here. The longer the label, the more space it needs in the flow. The shorter the label, the less information it can share.
+
+For some nodes, it might be appropriate to hide the label altogether to minimise the horizontal space it uses in the flow - giving more room to other nodes.
+
```

Comment:
maybe an image of the "appearance tab" in config to show how to get to shorten the name